src/main/java/org/alliancegenome/curation_api/services/Gff3Service.java

```diff
@@ -160,4 +174,74 @@
 		return idsAdded;
 	}
 	
+	@Transactional
+	public Map<String, List<Long>> loadParentChildAssociations(BulkLoadFileHistory history, ImmutablePair<Gff3DTO, Map<String,String>> gffEntryPair, Map<String, List<Long>> idsAdded, BackendBulkDataProvider dataProvider, String assemblyId, Map<String, String> geneIdCurieMap) throws ObjectUpdateException {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>',' is not followed by whitespace.
<!-- __reviewdog__:ChA0NDNiOGQ2M2U2ZTdjMTAxEgpjaGVja3N0eWxl -->